lib/ignored-paths.js

```diff
@@ -201,6 +201,12 @@ class IgnoredPaths {
 
         const ig = ignore().add(DEFAULT_IGNORE_DIRS);
 
+        if (this.options.dotfiles !== true) {
+
+            // Ignore hidden folders.  (This cannot be ".*", or else it's not possible to unignore hidden files)
+            ig.add([".*/*", "!../"]);
```

Comment:
I think it is not good to `ig.add('!../')`, because .gitignore only cares about the current directory and subtle directories.

Even if it works to do this, it is the side effect of `node-ignore` however.